src/main/java/de/tum/in/www1/artemis/repository/StatisticsRepository.java

```diff
@@ -27,6 +31,16 @@
             """)
     List<Map<String, Object>> getTotalSubmissions(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate);
 
+    @Query("""
+            select s.submissionDate as day, count(s.id) as amount
+            from Submission s
+            where s.submissionDate >= :#{#startDate} and s.submissionDate <= :#{#endDate} and s.participation.exercise.id in :exerciseIds
+            group by s.submissionDate
+            order by s.submissionDate asc
+            """)
+    List<Map<String, Object>> getTotalSubmissionsForCourse(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate,
```

Comment:
please use a custom object here and not a map

Reply:
Done